css/portal-dashboard/header.less

```diff
@@ -42,12 +42,205 @@
     justify-content: center;
     margin: 0 10px 0 10px;
   }
-
-  .login {
+  .headerRight{
     display: flex;
+    flex-direction: row;
+    align-items: center;
     flex: 1;
     justify-content: flex-end;
     margin: 0 10px 0 10px;
   }
+  .icon {
+    width: 32px;
+    height: 32px;
+    object-fit: contain;
+  }
+  .headerMenu {
+    height: 32px;
+    width: 32px;
+    padding: 12px;
+  }
+  .menuIcon {
+    fill: @cc-teal;
+    cursor: pointer;
+    transition-duration: 1.25s;
```

Comment:
I don't see it explicitly in the spec, but from using the prototype and implementing Michael's other specs, I bet this is supposed to be `transition-duration: .25s;`